### PR TITLE
Add uploadedAt to Package type

### PR DESCRIPTION
**Pull request by CalumMcCall**
- PRO-1963

### CODE REVIEW

File: core/src/main/resources/db/migration/V26__add_uploaded_date.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `Package`
+ADD COLUMN uploadedAt DATETIME(3) NOT NULL DEFAULT 0;
```
Comment (Danten) [inline core/src/main/resources/db/migration/V26__add_uploaded_date.sql]: Does 0 work as a default for dates?

Comment (CalumMcCall) [inline core/src/main/resources/db/migration/V26__add_uploaded_date.sql]: It's what we use for other DateTime columns. I'll double check they are instants.

File: core/src/main/scala/org/genivi/sota/core/db/Packages.scala [hunk 2]
```diff
@@ -50,14 +51,16 @@ object Packages {
...
 
     def uniqueUuid = index("Package_unique_name_version", (namespace, name, version), unique = true)
 
-    def * = (namespace, name, version, uri, size, checkSum, description.?, vendor.?, signature.?, uuid).shaped <>
-      (x => Package(x._1, x._10, PackageId(x._2, x._3), x._4, x._5, x._6, x._7, x._8, x._9),
-        (x: Package) => Some((x.namespace, x.id.name, x.id.version, x.uri, x.size, x.checkSum, x.description, x.vendor, x.signature, x.uuid)))
+    def * = (namespace, name, version, uri, size, checkSum, description.?, vendor.?, signature.?, uuid, uploadedAt).shaped <>
+      (x => Package(x._1, x._10, PackageId(x._2, x._3), x._4, x._5, x._6, x._7, x._8, x._9, x._11),
+        (x: Package) => Some((x.namespace, x.id.name, x.id.version, x.uri, x.size, x.checkSum, x.description, x
+          .vendor, x.signature, x.uuid, x.uploadedAt)))
```
Comment (Danten) [inline core/src/main/scala/org/genivi/sota/core/db/Packages.scala]: newline!!!

File: core/src/main/resources/db/migration/V26__add_uploaded_date.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `Package`
+ADD COLUMN uploadedAt DATETIME NOT NULL DEFAULT NOW();
```
Comment (jerrytrieu) [inline core/src/main/resources/db/migration/V26__add_uploaded_date.sql]: Actually, usual `createdAt`?

Comment (CalumMcCall) [inline core/src/main/resources/db/migration/V26__add_uploaded_date.sql]: Yeah, you're right. I'll change it.

**Pull request merged**